pkgs/applications/networking/seafile-client/default.nix

```
@@ -13,6 +13,15 @@ mkDerivation rec {
     sha256 = "2vV+6ZXjVg81JVLfWeD0UK+RdmpBxBU2Ozx790WFSyw=";
   };
 
+  patches = [
+    # Fix #122505
```

do you mind expanding on the nature of the patch?
```suggestion
    # Fix issue with <summary>, fixes #122505
```

---

Done